src/esssans/normalization.py

```diff
@@ -406,11 +406,7 @@ def _reduce(da: sc.DataArray) -> sc.DataArray:
                 )
     elif numerator.bins is not None:
         numerator = numerator.hist()
-    if numerator.bins is not None and denominator.bins is None:
-        da = numerator.bins / sc.lookup(func=denominator, dim='Q')
-    else:
-        da = numerator / denominator
-    return IofQ[RunType](da)
```

Comment:
Not sure I understand why we don't need this anymore?

Reply:
It was not actually related to this change, but since we, e.g., `bin(Q=...)` in `merge_spectra` we get the same normalization term for all events in a bin. The `lookup` was not actually required (or never), since we always performed such Q binning (or now: Qx, Qy binning).